### ISSUE TITLE
Is there api for adding chapter markers?

### ISSUE THREAD

**Issue by ruotianluo** [buzzsprout/buzzsprout-api#8]

**Comment by buzzsprout** [comment]
We don't have an endpoint to add chapter markers but I have added this as an enhancement.

**Comment by ruotianluo** [comment]
Looking forward to it.

**Comment by nnnnicholas** [comment]
@buzzsprout If we add chapter markers using a tool like https://github.com/jcs/mp3chap will the chapters be retained when uploaded to Buzzsprout?

**Comment by johnloringpollard** [comment]
Yes, we extract the chapter markers from the file and import them into our system.

**Comment by sumitbaggabhai** [comment]
How we can get channel details using channel ID?

**Comment by brydave** [comment]
Hey! This is available now.

You can list and replace an episode’s chapter markers via the API:
- Docs: https://github.com/buzzsprout/buzzsprout-api/blob/master/sections/chapters.md

Quick path:
1. `GET /api/:podcast_id/episodes/:episode_id/chapters`
2. Edit the list, then `PUT` the full replacement (empty array clears non-ad chapters)

Keep in mind, chapters embedded in an uploaded audio file are still extracted on upload, same as before.

Open a new issue if you need something this flow doesn’t support.
